webviz_subsurface/plugins/_co2_leakage/_plugin.py

```diff
@@ -427,31 +431,68 @@ def set_well_options(
-            State(self._view_component(MapViewElement.Ids.DECKGL_MAP), "views"),
+            inputs={
+                "attribute": Input(
+                    self._settings_component(ViewSettings.Ids.PROPERTY), "value"
+                ),
+                "date": Input(
+                    self._view_component(MapViewElement.Ids.DATE_SLIDER), "value"
+                ),
+                "formation": Input(
+                    self._settings_component(ViewSettings.Ids.FORMATION), "value"
+                ),
+                "realization": Input(
+                    self._settings_component(ViewSettings.Ids.REALIZATION), "value"
+                ),
+                "statistic": Input(
+                    self._settings_component(ViewSettings.Ids.STATISTIC), "value"
+                ),
+                "color_map_name": Input(
+                    self._settings_component(ViewSettings.Ids.COLOR_SCALE), "value"
+                ),
+                "cm_min_auto": Input(
+                    self._settings_component(ViewSettings.Ids.CM_MIN_AUTO), "value"
+                ),
+                "cm_min_val": Input(
+                    self._settings_component(ViewSettings.Ids.CM_MIN), "value"
+                ),
+                "cm_max_auto": Input(
+                    self._settings_component(ViewSettings.Ids.CM_MAX_AUTO), "value"
+                ),
+                "cm_max_val": Input(
+                    self._settings_component(ViewSettings.Ids.CM_MAX), "value"
+                ),
+                "plume_threshold": Input(
+                    self._settings_component(ViewSettings.Ids.PLUME_THRESHOLD), "value"
+                ),
+                "plume_smoothing": Input(
+                    self._settings_component(ViewSettings.Ids.PLUME_SMOOTHING), "value"
+                ),
+                "visualization_update": Input(
+                    self._settings_component(ViewSettings.Ids.VISUALIZATION_UPDATE),
+                    "n_clicks",
+                ),
+                "mass_unit": Input(
+                    self._settings_component(ViewSettings.Ids.MASS_UNIT), "value"
+                ),
+                "mass_unit_update": Input(
+                    self._settings_component(ViewSettings.Ids.MASS_UNIT_UPDATE),
+                    "n_clicks",
+                ),
+                "options_dialog_options": Input(
+                    ViewSettings.Ids.OPTIONS_DIALOG_OPTIONS, "value"
+                ),
+                "selected_wells": Input(
+                    ViewSettings.Ids.OPTIONS_DIALOG_WELL_FILTER, "value"
+                ),
+                "ensemble": Input(
+                    self._settings_component(ViewSettings.Ids.ENSEMBLE), "value"
+                ),
+                "current_views": State(
+                    self._view_component(MapViewElement.Ids.DECKGL_MAP), "views"
+                ),
+                "thresholds": [Input(id, "value") for id in self._threshold_ids],
+            },
```

Comment:
Er endringen til inputs = {} for å gjøre pylint fornøyd?

Reply:
Nei, er fordi thresholds, og dermed inputs totalt ikke har bestemt lengde.